ERCS/erc-7281.md

```diff
@@ -0,0 +1,356 @@
+---
+eip: 7281
+title: Sovereign Bridged Token
+description: An interface for creating fungible representations of tokens bridged across domains.
+author: Shaito (@0xShaito), Excalibor (@excaliborr), Arjun Bhuptani (@arjunbhuptani)
+discussions-to: https://ethereum-magicians.org/t/erc-7281-sovereign-bridged-tokens/14979
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-06-27
+requires: 20
+---
+
+## Abstract
+
+This proposal defines a minimal extension to [ERC-20](../EIPS/eip-20.md) (affectionately called `XERC20`) that enables bridging tokens across domains without creating multiple infungible representations of the same underlying asset. It introduces the concept of a Lockbox to allow existing tokens to comply with the specification through a familiar wrapping mechanism and exposes new interfaces that allow token issuers to apply custom risk profiles at a per bridge per domain granularity. 
```

Comment:
This is better! I'd still like to see a bit more detail on what a "Lockbox" is here, but you can work on that after you get merged as Draft.